cloud: add cloud release notes for release-20260519 (#22907)
pingcap/docs: tidb-cloud/releases/tidb-cloud-release-notes.md

This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2026.

## May 19, 2026

**General changes**

- **TiDB Cloud Essential**

>     - Top RU is now available in public preview for [TiDB Cloud Essential](/tidb-cloud/select-cluster-tier.md#essential) in the following regions:

Reviewer: ```suggestion
    - Top RU is now available in public preview for [TiDB Cloud Essential](/tidb-cloud/select-cluster-tier.md#essential) in the following region:
```

>         - AWS: `Oregon (us-west-2)`, `Frankfurt (eu-central-1)`, `Singapore (ap-southeast-1)`

Reviewer: ```suggestion
```

>         - Alibaba Cloud: Mexico (na-south-1)

Reviewer 2: ```suggestion
        - AWS: `Oregon (us-west-2)`, `Frankfurt (eu-central-1)`, `Singapore (ap-southeast-1)`
        - Alibaba Cloud: `Mexico (na-south-1)`
```

> 

Reviewer 2: For reviewer: for more information about region names, see https://www.pingcap.com/tidb-cloud-essential-pricing-details/

>       This feature displays minute-level top RU-consuming SQL statements, helping you quickly identify and optimize the most resource-intensive queries to reduce costs.

Reviewer: ```suggestion
      This feature displays minute-level top RU-consuming SQL statements, helping you quickly identify the most resource-intensive queries to reduce costs.
```

>       This feature shows the top RU-consuming SQL statements at minute-level granularity, helping you quickly identify and optimize resource-intensive queries to reduce costs.

Reviewer 2: ```suggestion
      This feature shows the top RU-consuming SQL statements at minute-level granularity, helping you quickly identify resource-intensive queries to reduce costs.
```

>         This feature shows the top RU-consuming SQL statements at minute-level granularity, helping you quickly identify and optimize resource-intensive queries to reduce costs.

Reviewer 2: ```suggestion
      This feature shows the top RU-consuming SQL statements at minute-level granularity, helping you quickly identify and optimize resource-intensive queries to reduce costs.
```

>       It is rolling out in phases. Contact our support team at [support@pingcap.com](mailto:support@pingcap.com) for early access.

Reviewer 2: ```suggestion
      This feature shows the top RU-consuming SQL statements at minute-level granularity, helping you quickly identify and optimize resource-intensive queries to reduce costs.

      The feature is rolling out in phases. To request early access, contact [support@pingcap.com](mailto:support@pingcap.com).
```

>       The feature is rolling out in phases. To request early access, contact [support@pingcap.com](mailto:support@pingcap.com).

Reviewer 2: ```suggestion
        This feature shows the top RU-consuming SQL statements at minute-level granularity, helping you quickly identify and optimize resource-intensive queries to reduce costs.

        The feature is rolling out in phases. To request early access, contact [support@pingcap.com](mailto:support@pingcap.com).
```
